target/linux/ipq806x/image/Makefile

```diff
@@ -284,6 +284,22 @@ define Device/netgear_r7800
 endef
 TARGET_DEVICES += netgear_r7800
 
+define Device/netgear_xr500
+	$(call Device/DniImage)
+	DEVICE_VENDOR := NETGEAR
+	DEVICE_MODEL := Nighthawk XR500
+	SOC := qcom-ipq8065
+	KERNEL_SIZE := 4096k
+	NETGEAR_BOARD_ID := XR500
+	NETGEAR_HW_ID := 29764958+0+256+512+4x4+4x4+cascade
+	BLOCKSIZE := 128k
+	PAGESIZE := 2048
+	BOARD_NAME := xr500
+	SUPPORTED_DEVICES += xr500
```

Comment:
Please remove BOARD_NAME and SUPPORTED_DEVICES.
I will have a look at the other stuff later.